### PR TITLE
feat: added share ask copy experiment

### PR DESCRIPTION
**Pull request by edwinf-barreram**
# Description
- Added new share ask copy experiment on thanks page.

# Ticket
[MARS-202](https://kiva.atlassian.net/browse/MARS-202)

**_Control_**

Screen Shot 2022-08-08 at 19 35 07

**_Variant_**

Variant

### CODE REVIEW

File: src/components/Thanks/ThanksPageShare.vue [hunk 1]
```diff
@@ -84,6 +84,15 @@
 							Introduce someone new to Kiva and we'll give you $25 to support another borrower. Your Kiva Lending Credit will be applied automatically.
 						</p>
 					</template>
+					<template v-else>
+						<h1	class="thanks__headline-h1 tw-mt-1 tw-mb-3 tw-text-left">
+							{{ this.lender.firstName }}, can you share this loan with one more person?
```
Comment (mcstover) [inline src/components/Thanks/ThanksPageShare.vue:89]: This name block needs to be masked:
```<span class="fs-mask">{{ this.lender.firstName }}</span>, can you share this loan with one more person?```

Comment (edwinf-barreram) [reply src/components/Thanks/ThanksPageShare.vue:89]: Will add it!

**Comment by mcstover** [comment]
@edwinf-barreram Line 16 in the ThanksPageShare.vue file needs to have the PII data masked as well. Please add the `fs-mask` class the strong tag around the email.
```<strong class="fs-mask">{{ lender.email }}</strong>```

**Review by mcstover — CHANGES_REQUESTED** [review]
Please mask the PII in these new interfaces.

File: src/components/Thanks/ThanksPageShare.vue [hunk 2]
```diff
@@ -84,6 +84,15 @@
 							Introduce someone new to Kiva and we'll give you $25 to support another borrower. Your Kiva Lending Credit will be applied automatically.
 						</p>
 					</template>
+					<template v-else>
+						<h1	class="thanks__headline-h1 tw-mt-1 tw-mb-3 tw-text-left">
+							<span class="fs-mask">{{ this.lender.firstName }}</span>, can you share this loan with one more person?
+						</h1>
+						<p class="tw-text-h3 tw-m-0 thanks__base-text">
+							<!-- eslint-disable-next-line max-len -->
+							{{ this.loan.name }} only needs {{ calculatePeopleQtyToGoal() }} more people to lend $25 and their loan could be fully funded in a matter of hours!
```
Comment (mcstover) [inline src/components/Thanks/ThanksPageShare.vue:93]: @edwinf-barreram Please mask the Borrower Name as well. It is masked in the original ThanksPage below already.

Comment (edwinf-barreram) [reply src/components/Thanks/ThanksPageShare.vue:93]: Done!

File: src/components/Thanks/ThanksPageShare.vue [hunk 3]
```diff
@@ -198,9 +209,14 @@ export default {
...
 			default: ''
-		}
+		},
+		shareAskCopyVersion: {
+			type: String,
+			default: 'a'
+		},
 	},
 	metaInfo() {
+		console.log('share ask copy', this.shareAskCopyVersion);
```
Comment (mcstover) [inline src/components/Thanks/ThanksPageShare.vue:219]: please remove

**Review by mcstover — APPROVED** [review]
Thanks for the updates!

**Review by dyersituations — APPROVED** [review]

File: src/graphql/query/thanksPage.graphql [hunk 4]
```diff
@@ -134,6 +134,12 @@ query checkoutReceipt($checkoutId: Int!, $visitorId: String) {
 			value
 		}
 	}
+	general {
+		shareCopyAsk: uiExperimentSetting(key: "share_copy_ask") {
```
Comment (emuvente) [inline src/graphql/query/thanksPage.graphql:138]: The key is supposed to be "share_ask_copy". I'll update it

**Review by mcstover — APPROVED** [review]

**Review by emuvente — APPROVED** [review]

**Pull request merged**